public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -50,5 +50,7 @@ enum class PurchasesErrorCode(val code: Int, val description: String) {
     LogOutWithAnonymousUserError(22, "Called logOut but the current user is anonymous."),
     ConfigurationError(23, "There is an issue with your configuration. Check the underlying error for more details."),
     UnsupportedError(24, "There was a problem with the operation. Looks like we doesn't support " +
-        "that yet. Check the underlying error for more details.")
+        "that yet. Check the underlying error for more details."),
+    UnknownHostError(25,  "It looks like requests to RevenueCat are being blocked. See: " +
+        "https://rev.cat/dnsBlocking for more info.")
```

Comment:
Looks like `https://rev.cat/dnsBlocking` isn't available yet?

Reply:
yeah, i just copied what was in ios.... but it's not. @taquitos do you know when that'll be up?

Reply:
I have it as a task on one of the Shortcuts, but it hasn't been started by anybody, yet.

Reply:
Oh, it's done now! https://rev.cat/dnsBlocking